docs/spec/enums.rst

```diff
@@ -123,8 +123,9 @@ statically in cases where dynamic values are used.
         genus: str  # Non-member attribute
         species: str  # Non-member attribute
 
-        CAT = ...  # Member attribute
-        DOG = ...  # Member attribute
+        CAT = 1  # Member attribute with known value and type
+        DOG = cast(str, ...)  # Member attribute with unknown value and known type
```

Comment:
As a reader, I think I would be confused by the use of `str` here, given that this appears to be a reimplementation of the above `Pet` enum. Why not just use `int` and keep things consistent?

Reply:
Changed it! I was anchored to some of the conversations we had around using `_value_` as a way to spell this (which would work for homogenous types, but not heterogeneous types)